diff --git a/src/app.module.ts b/src/app.module.ts
index b43b0d0..695fcc4 100644
--- a/src/app.module.ts
+++ b/src/app.module.ts
@@ -1,4 +1,5 @@
-import { Module } from '@nestjs/common';
+import { Module, ValidationPipe } from '@nestjs/common';
+import { APP_PIPE } from '@nestjs/core';
 import { HealthModule } from './health/health.module';
 import { CartModule } from './cart/cart.module';
 import { CheckoutModule } from './checkout/checkout.module';
@@ -6,5 +7,15 @@ import { ProductsModule } from './products/products.module';
 
 @Module({
   imports: [HealthModule, CartModule, CheckoutModule, ProductsModule],
+  providers: [
+    {
+      provide: APP_PIPE,
+      useValue: new ValidationPipe({
+        whitelist: true,
+        transform: true,
+        forbidNonWhitelisted: false,
+      }),
+    },
+  ],
 })
 export class AppModule {}
