diff --git a/src/checkout/checkout.service.ts b/src/checkout/checkout.service.ts
index 7d1efe0..5c9c3dc 100644
--- a/src/checkout/checkout.service.ts
+++ b/src/checkout/checkout.service.ts
@@ -36,7 +36,7 @@ export class CheckoutService {
       discount,
       tax,
       total,
-      formatedTotal: formatMoney(total),
+      formattedTotal: formatMoney(total),
       status: 'confirmed',
     };
   }
